PMM 4515 PMM-4525 Improving Description, Adding support for MongoDB official

puneet0191 opened the pull request.

rnovikovP commented on pmm/aws-staging-start.groovy:
```diff
@@ -47,9 +47,17 @@ pipeline {
+            description: '''
+            Configure PMM Clients
+            ms - MySQL (ex. --addclient=ms,1), To Start with Slowlog use (--addclient=ms,1 --query-source=slowlog),
+            ps - Percona Server for MySQL (ex. --addclient=ps,1),
```
Can we add an example here about Slow log. 
and for MariaDB too  ) 

thanks

puneet0191 replied:
I have added for all type, slowlog for Percona server will be added this week, I will fix it then

puneet0191 commented:
it is a lot of options and will make it complicated.

puneet0191 commented:
I have put the query-source as an option, so no need to worry about passing it as parameter, added example and also updated with MongoDB official support
